### DISCUSSION TITLE
Avidemux

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#18] [Requests] [answered]
Avidemux website: http://avidemux.sourceforge.net/

Avidemux privacy policy: https://github.com/mean00/avidemux2/blob/master/License.txt

Logo: https://upload.wikimedia.org/wikipedia/commons/d/d9/Avidemux-logo.png

Description: "Avidemux is a free and open-source software for non-linear video editing and transcoding multimedia files. The developers intend it as "a simple tool for simple video processing tasks" and to allow users "to do elementary things in a very straightforward way". It is written in C++ and uses Qt for its graphical user interface, and FFmpeg for its multimedia functions. Starting with version 2.4, Avidemux also offers a command-line interface, and since version 2.6, the original GTK port has not been maintained and is now discontinued.

Avidemux is developed for Linux, macOS, and Windows. Unofficial builds are also available for FreeBSD, NetBSD, and OpenBSD."

**Comment by mayrinck** [comment]
Nice Suggestion!

But the license is not the Privacy Policy, and they don't have the same meaning for Microsoft, so it won't work, just like antimicro... 
Name is reserved but we need that to progress further.

**Comment by mayrinck** [comment, marked as answer]
Just tried to convert the installation package, but it didn't work... So, sadly, I will drop this one too :/
